sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_automation_client_async.py

```diff
@@ -160,11 +167,21 @@ async def create_call(
         """Create a call connection request to a target identity.
 
         :param target_participant: Call invitee's information.
-        :type target_participant: ~azure.communication.callautomation.CallInvite
+        :type target_participant: ~azure.communication.callautomation.CommunicationIdentifier
```

Comment:
It does not match L155 

target_participant: Union['CommunicationIdentifier', List['CommunicationIdentifier']],

?